sdk/python/kfp/compiler/test_data/components/unsupported/output_named_tuple.py

```diff
@@ -22,7 +22,7 @@
 
```

Comment:
Does this file still belong in `unsupported/`? I believe this is now compilable and executable.

Reply:
Same consideration with `unsupported/train.py`

Reply:
`output_named_tuple` still belongs to `unsupported` as the name suggested. In short, our new artifact class types cannot be specified via return type annotation because the class object needs to be constructed outside (by `kfp.executor`) and then pass into the user function--that's how `artifacts.uri` is prefilled with backend generated URI value. The value is from the JSON known as "executor input" and resolved via the `{{$}}` (root placeholder).

Reply:
`unsupported/train.py` combines and repeats unsupported feature from `artifact_consumer.py` and `output_named_tuple.py`. Looks like some dupe to me.

Reply:
Thank you for explaining. That makes sense and agreed that this is still unsupported.

Re: dups, also agreed